spaces/S000198/properties/P000097.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000198
+property: P000097 # Embeddable in the Reals
+value: true
+---
+
+This space is a copy of $(0,1)\cup\{2\}$.
```

Comment:
Same thing, but could we say: "This space is homeomorphic to ..." ?

Reply:
```suggestion
This space is homeomorphic to $(0,1)\cup\{2\}$.
```